<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-12-19T23:26:26 IST" name="Suite" finished-at="2024-12-19T23:27:00 IST" duration-ms="33866">
    <groups>
    </groups>
    <test started-at="2024-12-19T23:26:26 IST" name="Test" finished-at="2024-12-19T23:27:00 IST" duration-ms="33866">
      <class name="testCases.TC103_ContactAdditionTest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC103_ContactAdditionTest@21a21c64]" started-at="2024-12-19T23:26:26 IST" name="setup" finished-at="2024-12-19T23:26:29 IST" duration-ms="2838" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testContactListMandatoryfileds()[pri:1, instance:testCases.TC103_ContactAdditionTest@21a21c64]" started-at="2024-12-19T23:26:29 IST" name="testContactListMandatoryfileds" finished-at="2024-12-19T23:26:41 IST" duration-ms="12211" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testContactListMandatoryfileds -->
        <test-method signature="testContactListoptionalfileds()[pri:2, instance:testCases.TC103_ContactAdditionTest@21a21c64]" started-at="2024-12-19T23:26:41 IST" name="testContactListoptionalfileds" finished-at="2024-12-19T23:26:48 IST" duration-ms="6903" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testContactListoptionalfileds -->
        <test-method signature="duplicateContactList()[pri:3, instance:testCases.TC103_ContactAdditionTest@21a21c64]" started-at="2024-12-19T23:26:48 IST" name="duplicateContactList" finished-at="2024-12-19T23:27:00 IST" duration-ms="11283" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Expected error message for duplicate contact, but got:  expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Expected error message for duplicate contact, but got:  expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at testCases.TC103_ContactAdditionTest.duplicateContactList(TC103_ContactAdditionTest.java:97)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- duplicateContactList -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:testCases.TC103_ContactAdditionTest@21a21c64]" started-at="2024-12-19T23:27:00 IST" name="teardown" finished-at="2024-12-19T23:27:00 IST" duration-ms="269" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- testCases.TC103_ContactAdditionTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
